qubrabench/benchmark.py

```diff
@@ -227,6 +245,10 @@ def combine_subroutine_frames(frames: list[BenchmarkFrame]) -> BenchmarkFrame:
                 quantum_expected_quantum_queries=max(
                     stats.quantum_expected_quantum_queries for stats in sub_frame_stats
                 ),
+                quantum_worst_case_classical_queries=0,
+                quantum_worst_case_quantum_queries=max(
+                    stats.quantum_worst_case_total_queries for stats in sub_frame_stats
+                ),
```

Comment:
Is there a factor 2 missing here?

Reply:
No, this entry is the max number of queries to implement the unitary computing all values of e.g. `key` for search (allowing use of workspace registers).

The 2x factor can be found in `search` (called `c_q`) whenever this unitary is used by Grover.